### ISSUE TITLE
ML-Benchmark prediction submission: FlowMatching-v1

### ISSUE THREAD

**Issue by mo-tomaswetherell** [WCRP-CORDEX/ml-benchmark#37]
### Institution ID

MO

### Institution Name

Met Office 

### Emulator identifier (`emulator_id`)

FlowMatching-v1

### Emulator Description

Flow matching model, with a U-Net neural network. 

**Predictors**
u, v, q, t, z at multiple pressure levels, normalised with z-score normalisation. 

**Targets**
pr or tasmax; pr normalised with log1p + z-score normalisation, tasmax with z-score normalisation.

**Training configuration**
Adam optimiser.

```
training:
  learning_rate: 0.0001
  epochs: 50
  batch_size: 64
  patience: 10
  years:
    Emulator_hist_future:
      [
        "1961-1966",
        "1968-1974",
        "1976-1980",
        "2080-2086",
        "2088-2094",
        "2096-2099",
      ]
    ESD_pseudo_reality: ["1961-1978"]

validation:
  years:
    Emulator_hist_future: ["1967", "1975", "2087", "2095"]
    ESD_pseudo_reality: ["1979-1980"]
```

**Network configuration**
ADM U-net with:

```
model:
  img_resolution: 128
  model_channels: 128
  channel_mult: [1, 2, 2, 2, 2]
  channel_mult_emb: 2
  num_residual_blocks: 2
  attention_resolutions: [8]
  dropout: 0.1
```

**Flow Matching Hyperparameters**

```
scheduler:
  type: condot

source:
  type: gaussian
  mean: 0.0
  std: 1.0
```

**Other**

Seperate models were trained for predicting precipitation and temperature.

### Hardware and Training Details

The models were trained on a single H100 node (94 GB VRAM). 

Each model took ~45 minutes to train (50 epochs, so ~54 seconds per epoch).  

Inference took 2-3 minutes per year of predictions (with a batch size of 256, which used 52 GB of VRAM).

### Stochastic/Probabilistic Output

yes

### Reference URL

https://github.com/mo-tomaswetherell/mo-flow-cordexbench

### Repository for Reproducibility

https://github.com/mo-tomaswetherell/mo-flow-cordexbench
